hnn_core/network.py

```diff
@@ -734,9 +750,11 @@ def _attach_drive(self, name, drive, weights_ampa, weights_nmda, location,
         drive['name'] = name  # for easier for-looping later
         drive['target_types'] = target_populations  # for _connect_celltypes
 
+        scaled_space_constant = space_constant * self._inplane_distance
```

Comment:
I think this could be surprising to the user who provided a certain `lamtha` through the drives API and ultimately it was scaled and a different one was used.

Reply:
Very good point!